src/kiwixapp.cpp

```diff
@@ -507,18 +508,40 @@ void KiwixApp::postInit() {
 
 void KiwixApp::handleItemsState(TabType tabType)
 {
-    auto libraryOrSettingsTab =  (tabType == TabType::LibraryTab || tabType == TabType::SettingsTab);
+    auto libraryOrSettingsTab = (tabType == TabType::LibraryTab || tabType == TabType::SettingsTab);
     auto notBookmarkableTab = libraryOrSettingsTab || getTabWidget()->currentArticleUrl().isEmpty();
+    auto hasZimFile = !getTabWidget()->currentZimId().isEmpty();
     auto app = KiwixApp::instance();
+
+    // Navigation actions
     app->getAction(KiwixApp::ToggleTOCAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::OpenHomePageAction)->setDisabled(!hasZimFile);
+    app->getAction(KiwixApp::RandomArticleAction)->setDisabled(!hasZimFile);
+
+    // Reading & bookmarks
     app->getAction(KiwixApp::ToggleReadingListAction)->setDisabled(libraryOrSettingsTab);
     app->getAction(KiwixApp::ToggleAddBookmarkAction)->setDisabled(notBookmarkableTab);
+    app->getAction(KiwixApp::ReadArticleAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::ReadTextAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::ReadStopAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::ToggleTTSLanguageAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::ToggleTTSVoiceAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::IncreaseTTSSpeedAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::DecreaseTTSSpeedAction)->setDisabled(libraryOrSettingsTab);
+
+    // Search & zoom
     app->getAction(KiwixApp::FindInPageAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::SearchArticleAction)->setDisabled(libraryOrSettingsTab);
     app->getAction(KiwixApp::ZoomInAction)->setDisabled(libraryOrSettingsTab);
     app->getAction(KiwixApp::ZoomOutAction)->setDisabled(libraryOrSettingsTab);
     app->getAction(KiwixApp::ZoomResetAction)->setDisabled(libraryOrSettingsTab);
-    app->getAction(KiwixApp::RandomArticleAction)->setDisabled(libraryOrSettingsTab);
-    app->getAction(KiwixApp::OpenHomePageAction)->setDisabled(libraryOrSettingsTab);
+
+    // File operations
+    app->getAction(KiwixApp::PrintAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::SavePageAsAction)->setDisabled(libraryOrSettingsTab);
+    app->getAction(KiwixApp::OpenMultiZimAction)->setDisabled(!hasZimFile);
+    app->getAction(KiwixApp::ExportReadingListAction)->setDisabled(!hasZimFile);
+    app->getAction(KiwixApp::ImportReadingListAction)->setDisabled(!hasZimFile);
```

Comment:
None of these actions is directly related to the ZIM file associated with the current tab.
- Multi-ZIM search options (yes, the action name is quite misleading - it is not a file operation and should probably be moved to a different group) should work on any tab (so that you can search in any ZIM file without first having to open that ZIM file in a tab);
- Bookmarks (reading-list) sidebar is displayed only in a ZIM tab, however the reading list is not bound to any particular ZIM file.